packages/governance/src/types.js

```diff
@@ -539,7 +544,7 @@
  * @typedef {Object} GovernedPublicFacet
  * @property {() => Subscription<ParamDescription>} getSubscription
  * @property {VoteOnParamChange} getContractGovernor
- * @property {GetParams} getGovernedParams - get descriptions of
+ * @property {GetGovernedVaultParams} getGovernedParams - get descriptions of
```

Comment:
@turadg 

It may be the case that the only actual use of this is in Vaults, but the support in Governance should be more generic. Is there a way to have a generic GetParams here, and override it with something more specific in Vaults?

Reply:
Yes, if in the vaults we use something more specific instead of `GovernedPublicFacet`. Happy to work together on the specifics.